src/PaymentGateways/Gateways/TestGateway/TestGateway.php

```diff
@@ -43,6 +44,15 @@ public function getName(): string
         return __('Test Gateway', 'give');
     }
 
+    /**
+     * @unreleased
+     */
+    public function enqueueScript(int $formId)
+    {
+        // temporary action to enqueue gateway scripts in feature plugin
```

Comment:
So is this going to be removed when the Feature Plugin is merged? If so, let's expand this out to explain that and leave a TODO.

Reply:
Yeah we need to setup wp scripts in givewp core so I just want to do that seperately, i'll make a note

Reply:
resolved f4f97558